pkg/services/sqlstore/user.go

```diff
@@ -473,6 +475,23 @@ func SearchUsers(query *m.SearchUsersQuery) error {
 	return err
 }
 
+func DisableUser(cmd *m.DisableUserCommand) error {
+	return inTransaction(func(sess *DBSession) error {
```

Comment:
This is the only one SQL statement right? If so, why we are in transaction?

Reply:
Hm, this looks redundant, but I just looked at the other functions that change DB data and found that it use transactions :)

Reply:
@xlson what do you think? I just tried to use the same code style, but it really looks strange. Should I use `inTransaction()` for a single DB query?

Reply:
I don't think there's any reason to use a transaction here. I'd just remove it for now.